docs/.ruby-version

```diff
@@ -0,0 +1 @@
+3.0
```

Comment:
I did not see such a file on https://github.com/nmstate/nmstate.github.io, is this a must?

Reply:
The are using github own jekyll version (which is very limiting it's jekyll < 4.0) so they don't have the infra.

Reply:
It allow us to have diagrams with mermeid and more stuff, is similar to the stuff they have a kubevirt.io maintenance is not a problem at kubernetes-nmstate is using the same set of configuration and we didn't need to to spend much time on maintaining it, updating the deps is similar to golang with `make update`.

Reply:
Well with jekyll github-pages you have similar issues except you cannot control it or if you find a bug at jekyll that was fixed you have to live with it, since we have already see at kubernetes-nmstate that is not of a burden I think we can live with it, but let's wait for @AlonaKaplan opinion.

Reply:
I understhand the point, the Gems list is the scary part but from the experience the kubernetes-nmstate you don't have to update it too much and since they are pinned they don't disappear, also the Makefile is quite clear on the purpose of each target. I will try to add a README.md to the docs directory to help with docs maintain duties.